src/test/java/plugins/GitPluginTest.java

```diff
@@ -127,10 +127,10 @@ public void name_remote_repo() {
         job.addShellStep("test -f foo && git remote -v");
         job.save();
 
-        job.startBuild().shouldSucceed().shouldContainsConsoleOutput("custom_origin\\s+" + repoUrl);
+        job.startBuild().shouldSucceed().shouldContainsConsoleOutput("custom_origin\\s+" + 
+                repoUrl.replace("[", "\\[").replace("]", "\\]"));
```

Comment:
use [`Pattern.quote`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/regex/Pattern.html#quote(java.lang.String))  host could contain other characters that are special (ie a `.` should be treated as a fullstop not any character)

Reply:
Done in https://github.com/jenkinsci/acceptance-test-harness/pull/1510/commits/40e880730b2f0b251f11049455f3df2ffd30a370